instance/models/instance.py

```diff
@@ -204,7 +204,7 @@ class GitHubInstanceMixin(VersionControlInstanceMixin):
     """
     github_organization_name = models.CharField(max_length=200, db_index=True)
     github_repository_name = models.CharField(max_length=200, db_index=True)
-    github_pr_number = models.IntegerField(blank=True, null=True)
+    github_pr_url = models.CharField(max_length=200, blank=True, default='')
```

Comment:
@Kelketek Shouldn't this be a URL field? Also the `default=''` is superflous I think - Django might ask for a default value when creating the migrations, but you can put `''` there rather than here?